tests/unit/query/indexeddb-adapter-test.js

```diff
@@ -645,6 +669,30 @@ test('adapter | indexeddb | false predicate | detail predicate', (assert) => {
   });
 });
 
+test('adapter | indexeddb | false predicate | detail predicate', (assert) => {
```

Comment:
Для предыдущего теста можно тоже имя уточнить.
```suggestion
test('adapter | indexeddb | false predicate | detail predicate all', (assert) => {
```